src/test/java/org/jenkinsci/plugins/workflow/support/steps/build/BuildTriggerStepTest.java

```diff
@@ -430,6 +450,23 @@ public MultiBranchProjectFactory newInstance() {
         }
     }
 
+    private void assertBuildInfoAction(WorkflowRun workflowRun, String projectName, int buildNumber) {
+        FlowGraphWalker walker = new FlowGraphWalker(workflowRun.getExecution());
```

Comment:
It needs `workflow-api:2.17`, so, I updated it in `pom.xml` as well, but now it fails in verification: https://ci.jenkins.io/blue/organizations/jenkins/Plugins%2Fpipeline-build-step-plugin/detail/PR-13/9/tests/
Also, I think @jglick would not be happy to introduce dependency for higher version (especially in case of test code needs it -> not production).

So, what should I do?